README.md

```diff
@@ -168,30 +169,30 @@ for grabber in grabbers.values():
-|----------------------------|-----------------|------------|-----------|-----------|-----------|-----------|-----------|-----------|
-| name                       | On Robot Arm    | optional   | optional  | optional  | optional  | optional  | optional | optional |
-| input_type                 | generic_usb    | required   | required  | required  | required  | required  | required | required |
-| id.serial_number           | 23458234       | optional   | -         | optional  | optional  | -  | - | - |
-| id.rtsp_url                | rtsp://…        | -          | required  | -         | -         | -         | - | - |
-| id.hls_url                 | https://.../*.m3u8     | -          | -         | -         | -         | -         | required | - |
-| id.youtube_url             | https://www.youtube.com/watch?v=...     | -          | -         | -         | -         | -         | - | required |
-| options.resolution.height  | 480            | optional   | -         | -         | optional  | -  | - | - |
-| options.resolution.width   | 640            | optional   | -         | -         | optional  | -  | - | - |
-| options.zoom.digital       | 1.3            | optional   | optional  | optional  | optional  | optional  | optional | optional |
-| options.crop.pixels.top    | 100            | optional   | optional  | optional  | optional  | optional  | optional | optional |
-| options.crop.pixels.bottom | 400            | optional   | optional  | optional  | optional  | optional  | optional | optional |
-| options.crop.pixels.left   | 100            | optional   | optional  | optional  | optional  | optional  | optional | optional |
-| options.crop.pixels.right  | 400            | optional   | optional  | optional  | optional  | optional  | optional | optional |
-| options.crop.relative.top  | 0.1            | optional   | optional  | optional  | optional  | optional  | optional | optional |
-| options.crop.relative.bottom | 0.9          | optional   | optional  | optional  | optional  | optional  | optional | optional |
-| options.crop.relative.left | 0.1            | optional   | optional  | optional  | optional  | optional  | optional | optional |
-| options.crop.relative.right | 0.9            | optional   | optional  | optional  | optional  | optional  | optional | optional |
-| options.depth.side_by_side | 1              | -          | -         | -         | optional  | -  | - | - |
-| options.num_90_deg_rotations | 2              | optional          | optional         | optional         | optional  | optional  | optional | optional |
-| options.keep_connection_open | True              | -          | optional         | -         | -  | - | optional | optional |
-| options.max_fps | 30              | -          | optional         | -         | -  | - | - | - |
-
+| Configuration Name         | Example         | Generic USB     | RTSP      | Basler    | Realsense | Raspberry Pi CSI2 | HLS | YouTube Live | File Stream |
```

Comment:
This table is becoming a little hard to read on github.com due to the need for horizontal scrolling. Not a huge deal, but a little annoying. Not sure to fix this aside from redoing it in html/css.